docs/blog/20221215-confused-Techie-v1.100.1-beta.md

```diff
@@ -0,0 +1,203 @@
+---
+title: Our First Release!
+author: confused-Techie
+date: 2022-12-15
+category:
+  - dev
+tag:
+  - sunset
+---
+
+Check out our first GitHub Release for Pulsar! [Available Now!](https://github.com/pulsar-edit/pulsar/releases/tag/v1.100.1-beta)
```

Comment:
We can change this Release URL from pointing to the `v1.100.1-beta` tag to --> `v1.100.0-beta` (.**1** --> .**0**).

Also I guess we could update the filename of this blog post accordingly, bu maybe folks have already linked to this, in which case we would probably not want to break those links in the wild.

I can do a quick PR for the URL i the blog post if you want. And changing the filename _if_ that seems worth it, or just leave it as-is as a historical quirk preserved in time as of when the post went live.